openmdao/recorders/__init__.py

```diff
@@ -0,0 +1,6 @@
+# <<<<<<< HEAD
+# from openmdao.recorders.base_recorder import BaseRecorder
```

Comment:
merge conflict? I guess it is commented out, but maybe we should clean it all out